### PR TITLE
Short name and alternatives

### PR DESCRIPTION
**Pull request by gkellogg**
This updates references to "rch-rdc" and "rdf-dataset-canonicalization" and adds @yamdan as an editor. It also changes a paragraph in SOTD to reference the issues related to the future selection of an algorithm (word-smithing appreciated).

Note, @yamdan, the editor entry could use information about you including, a personal URL, affiliation, affiliation URL, and most importantly, your w3cid, which I believe is required for publication. @pchampin can probably help in finding this.

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/rdf-canon/pull/28.html" title="Last updated on Nov 10, 2022, 6:02 PM UTC (573e9e2)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/rdf-canon/28/50485e3...573e9e2.html" title="Last updated on Nov 10, 2022, 6:02 PM UTC (573e9e2)">Diff</a>

### CODE REVIEW

**Comment by gkellogg** [comment]
Also, note that the old spec link https://w3c.github.io/rch-rdc/spec/ does not properly redirect to the new location https://w3c.github.io/rdf-canon/spec/.

**Comment by davidlehn** [comment]
@gkellogg: should be @yamdan in above comment.  correct in the CODEOWNERS but note the error that @yamdan needs write access to repo.

File: tests/README.md
Comment (davidlehn) [inline tests/README.md:10]:
```suggestion
[https://github.com/w3c/rdf-canon](https://github.com/w3c/rdf-canon)
```

**Review by dlongley — APPROVED** [review]
LGTM, thanks!

File: spec/index.html
Comment (iherman) [inline spec/index.html:50]:
```suggestion
      { name:       "Dan Yamamoto" 
        w3cid:       139477}
```

**Review by iherman — APPROVED** [review]

File: spec/index.html
Comment (gkellogg) [reply spec/index.html:50]: Missing comma, but thanks.

Comment (yamdan) [reply spec/index.html:50]: Thank you @iherman for adding my w3cid.

It seems to me that I should not add my company info here because my employer is currently not a W3C member.
Should I add my github URL "https://github.com/yamdan" here? That is my only "personal" URL...

Comment (gkellogg) [reply spec/index.html:50]: We can use your GitHub url,p.

File: tests/manifest-urdna2015.jsonld
Comment (yamdan) [inline tests/manifest-urdna2015.jsonld:7]:
```suggestion
    "rdfn": "http://w3c.github.io/rdf-canon/tests/vocab#",
```

File: tests/mk_manifest.rb
Comment (yamdan) [inline tests/mk_manifest.rb:73]:
```suggestion
      "rdfn": "http://w3c.github.io/rdf-canon/tests/vocab#",
```

**Review by yamdan — CHANGES_REQUESTED** [review]
If I'm not mistaken, the test vocabulary URI should start with `http` instead of `https`.

File: tests/manifest-urdna2015.jsonld
Comment (peacekeeper) [reply tests/manifest-urdna2015.jsonld:7]: I guess it doesn't really matter, but I wonder if there is some "best practice" in the RDF / JSON-LD world whether to use http or https for namespaces..

**Review by peacekeeper — APPROVED** [review]
Maybe one of our RDF gurus could comment on @yamdan 's suggestion to use http for RDF namespaces, otherwise all looks good. Thanks @gkellogg for adding the note about the other algorithm!

File: spec/index.html
Comment (iherman) [reply spec/index.html:50]: Indeed, not your company info. The github URL should be o.k.

**Comment by iherman** [comment]
I am not really an RDF guru, I leave this to @pchampin, but I do not see any reason why not using https. The "traditional" URL-s (for rdf, rdfs, owl, xsd, etc.) that were defined before https became widespread must indeed remain as they are to ensure backward compatibility. That is mostly true for schema.org as well (although google does make the same vocabulary available via https). But for newer vocabularies (e.g., the credentials vocabulary) https is probably better; simply because there is a general push to use https instead of http on the web in general. This is also true for our 'own' vocabulary for testing.

@msporny and @dlongley had some discussions on that, I presume, at the time of the early VC versions, maybe they have something to add.

cc @pchampin

**Comment by yamdan** [comment]
I apologize if my wording was not clear and misleading, I did not suggest to use `http` URI according to best practices, but only to be consistent with other test data (e.g., tests/manifest-urdna2015.ttl, test/mk_vocab.rb, ...) and the vocabulary URI definition in [RDF Dataset Canonicalization Test Vocabulary](https://w3c.github.io/rdf-canon/tests/vocab#).
>  The URI of the vocabulary is **http**://w3c.github.io/rch-rdc/tests/vocab# (abbreviated by rdfn in this document).

File: spec/index.html
Comment (yamdan) [reply spec/index.html:50]: Thanks, then it would look something like this:
```suggestion
      { name:       "Dan Yamamoto",
        url:        "https://github.com/yamdan",
        w3cid:      "139477" }
```

**Comment by msporny** [comment]
> @msporny and @dlongley had some discussions on that, I presume, at the time of the early VC versions, maybe they have something to add.

We should be using `https` everywhere. There is only ONE good reason to NOT do this today and that's that it breaks software in production that can't be upgraded (for legitimate reasons). I don't think we're in that situation here, certainly all new vocabulary URLs MUST be served via `https`. Hope that helps. :)

**Comment by gkellogg** [comment]
> I apologize if my wording was not clear and misleading, I did not suggest to use `http` URI according to "best practices", but only to be consistent with other test data (e.g., tests/manifest-urdna2015.ttl, test/mk_vocab.rb, ...) and the vocabulary URI definition in [RDF Dataset Canonicalization Test Vocabulary](https://w3c.github.io/rdf-canon/tests/vocab#).
> 
> > The URI of the vocabulary is **http**://w3c.github.io/rch-rdc/tests/vocab# (abbreviated by rdfn in this document).

If we were using a pre-existing vocabulary URI, we could not change from “http” to “https”, but for a new namespace, I think it’s a best practice to use “https”. IMHO, it was a mistake to introduce a new scheme for secure HTTP, but that’s what we have to deal with.

**Comment by yamdan** [comment]
Thank you for your comments about a current best practice to use `https` for vocabulary URIs. I open a PR #30 to reflect this practice to our test cases, only for our test vocabulary `rdfn:`.

**Pull request merged**
